### PR TITLE
Feat/node lambda docs

### PR DESCRIPTION
**Pull request by ajjindal**
Latest Node AWS integration docs update

### CODE REVIEW

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 1]
```diff
@@ -4,6 +4,9 @@ redirect_from:
   - /platforms/node/aws_lambda/
 ---
 
+*Verify Version*
+_(New in version 5.22.4 - Lambda integration is in Beta)_
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:8]: It'll be `5.23.0`

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 2]
```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
...
-To set up Sentry error logging for a Lambda Function, build a wrapper:
+You can use the AWS Lambda integration for the Node SDK like this:
 
 ```javascript
-"use strict";
+const Sentry = require("@sentry/serverless");
+
+Sentry.init({
+  dsn:
+    "___PUBLIC_DSN___",
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:29]:
```suggestion
  dsn: "___PUBLIC_DSN___"
```

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 3]
```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
 $ yarn add @sentry/node
 ```
 
-To set up Sentry error logging for a Lambda Function, build a wrapper:
+You can use the AWS Lambda integration for the Node SDK like this:
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:22]: I'd remove the `SDK` part from the `Node SDK`

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 4]
```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
...
 ```javascript
-"use strict";
+const Sentry = require("@sentry/serverless");
+
+Sentry.init({
+  dsn:
+    "___PUBLIC_DSN___",
+});
+
+const myAsyncHandler = async (event, context, callback) => {
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:32]:
```suggestion
const myAsyncHandler = async (event, context) => {
```

`async` functions are mutually exclusive with `callback`. It's either async or sync with callback.

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 5]
```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
...
+
+exports.handler = Sentry.AWSLambda.wrapHandler(myAsyncHandler);
+```
+
+<!-- TODO-ADD-VERIFICATION-EXAMPLE -->
 
-const Sentry = require("@sentry/node");
+## Timeout Warning
 
+Sentry reports timeout warning when the function is within 500ms of it's execution time. You can turn off timeout warnings by setting `captureTimeoutWarning` to `false` during initiatlization. To change timeout warning limit, assign a numeric value (in ms) to `timeoutWarning`.
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:43]: This is now how options are set for handlers.

```js
Sentry.AWSLambda.wrapHandler(yourHandler, {
  captureTimeoutWarning: false
});
```

So both, description and code examples are incorrect.

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 6]
```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
...
+```javascript {tabTitle:timeoutWarning}
+Sentry.init({
+  dsn:
+    "___PUBLIC_DSN___",
+    timeoutWarning: 50
 });
 ```
 
-You can obtain the DSN using your Sentry account from your organization's _Settings > Projects > Client Keys (DSN)_ in the Sentry web UI.
+The timeout warning is sent only if the "timeout" in the Lambda Function configuration is set to a value greater than one second.
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:61]: There's nothing stopping you from setting timeout to `1s` and still capture the warning if limit is small enough.

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 7]
```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
...
-Note: You need to call both `captureException` and `flush` for captured events to be successfully delivered to Sentry.
+  - A link to the cloudwatch logs
+  - Function details
+  - sys.argv for the function
+  - AWS Request ID
+  - Function execution time
+  - Function version
 
-Create the deployment package in `.zip` format, then upload it to AWS Lambda as a Lambda Function. Checkout Sentry's [aws sample apps](https://github.com/getsentry/examples/tree/master/aws-lambda/node) for detailed examples. Refer to the [JavaScript docs](/platforms/javascript/) for more configuration options.
+- Sentry holds the thread for upto 2 seconds to report errors. You can change flush time limit by defining a `flushTimeout` value during initiatlization
```
Comment (kamilogorek) [inline src/platforms/node/guides/aws-lambda/index.mdx:78]: in the handler options*

File: src/wizard/node/awslambda.md [hunk 8]
```diff
@@ -16,33 +17,43 @@ $ npm install --save @sentry/node
 $ yarn add @sentry/node
 ```
 
-To set up Sentry error logging for a Lambda Function, build a wrapper:
+You can use the AWS Lambda integration for the Node SDK like this:
```
Comment (kamilogorek) [inline src/wizard/node/awslambda.md:20]: All the comments apply to this file as well, so I won't double-post them.

**Review by kamilogorek — CHANGES_REQUESTED** [review]

**Comment by HazAT** [comment]
Outdated

**Pull request closed without merging**